pkg/testutils/ingestor/k8s.yaml

```diff
@@ -115,7 +115,6 @@ spec:
           command:
             - /ingestor
           args:
-            - "--kustainer"
             - "--storage-dir=/mnt/data"
             - "--max-segment-age=5s"
             - "--max-disk-usage=21474836480"
```

Comment:
Specifying that we're talking to Kustainer is no longer necessary. Instead, we infer the capabilities of a Kusto cluster by inspecting the cluster's details.